docs/services/compute/virtual-machines.md

```diff
@@ -42,221 +42,337 @@ An Azure virtual machine gives you the flexibility of virtualization without hav
 
 
-1. **Utilize Azure Monitor:**
-   - Leverage Azure Monitor to access detailed insights into VM performance metrics. Set up alerts for key metrics such as CPU usage, memory utilization, and disk activity.
+1. **Utilize Azure monitor:**
+   - Leverage Azure monitor to access detailed insights into VM performance metrics. Set up alerts for key metrics such as CPU usage, memory utilization, and disk activity.
 
-2. **Cost Management Tools:**
+2. **Cost management tools:**
    - Leverage the cost management tools, such as Azure Cost Management and Power BI, to analyze spending patterns and detect unusual spikes in VM costs.
 
-3. **Performance Metrics:**
+3. **Performance metrics:**
    - Correlate cost anomalies with performance metrics. Anomalies in resource utilization may directly impact costs.
 
-4. **Resource Configuration:**
+4. **Resource configuration:**
    - Evaluate the configuration of VM resources. Anomalies could be linked to improper resource allocation or sudden changes in demand.
 
 <br>
 
-## Optimize cloud usage and cost
+## Quantify business value
 
 ### Unit economics
 
+<!---
+Add link to MS or F2 doc explaining the capability
+-->
+
 
 ### Estimation + forecasting
+<!---
+Add link to MS or F2 doc explaining the capability
+-->
 
 Before creating a VM, leverage the [Azure Pricing Calculator](https://azure.microsoft.com/en-us/pricing/calculator/) to simulate costs. This tool allows you to estimate expenses based on your selected VM configurations, helping you make informed decisions and plan budgets effectively.
 
-Understanding and proactively managing the costs associated with virtual machines and related services are key components of a successful cloud financial management strategy. Regularly review and optimize your resource usage to ensure cost-effectiveness and alignment with your business goals.
+Understanding and proactively managing the costs associated with VMs and related services are key components of a successful cloud financial management strategy. Regularly review and optimize your resource usage to ensure cost-effectiveness and alignment with your business goals.
 
 ### Budgeting
-Effectively budgeting for Virtual Machines (VMs) in Azure involves considersing different factors, including VM sizes, usage patterns, and potential discounts.
+<!---
+Add link to MS or F2 doc explaining the capability
+-->
+
+Effectively budgeting for VMs in Azure involves considersing different factors, including VM sizes, usage patterns, and potential discounts.
+
+
+
+### Benchmarking
+<!---
+Add link to MS or F2 doc explaining the capability
+-->
+<br>
+
+## Quantify business value
+<!---
+Add link to MS or F2 doc explaining the capability
+-->
+
+### Architecting for cloud
+
+<!---
+Add link to MS or F2 doc explaining the capability
+-->
 
-## Understanding VM Sizes (SKUs):
+## Understanding VM sizes (SKUs):
```

Comment:
Do these headers add any value? Could we just keep this as a simple bulleted list?

Side note: Don't add an H2 under an H3 😜